### ISSUE TITLE
Files not properly processed: in Finalization but broken

### ISSUE THREAD

**Issue by jcowey** [papyri/sosol#100]
Similar to #95, #99. 
Can you please make this file available for finalization (they are in finalization but broken):

 - [x] http://papyri.info/editor/publications/62730/ddb_identifiers/141371/edit

Can you please keep it assigned to me

http://papyri.info/editor/users/james.cowey

Thanks

**Comment by ryanfb** [comment]
I've re-sent this to you for finalization at http://papyri.info/editor/publications/62927. Please re-open with more information about what's going wrong if there are still problems.

**Comment by jcowey** [comment]
OK. If I open this at:
http://papyri.info/editor/publications/62927
everything seems to be in good order.
Now I click on 
http://papyri.info/editor/publications/62927/ddb_identifiers/142266/edit
and receive the following page:
![bildschirmfoto 2017-12-20 um 16 15 23](https://user-images.githubusercontent.com/612992/34213928-0c75ce76-e5a1-11e7-8a94-6b663676c91e.png)

This is what the XML page looks like:
http://papyri.info/editor/publications/62927/ddb_identifiers/142266/editxml

![bildschirmfoto 2017-12-20 um 16 16 06](https://user-images.githubusercontent.com/612992/34213961-2fb61b5c-e5a1-11e7-8369-87e01f04f828.png)

in other words apparently empty.

When I click on 
http://papyri.info/editor/publications/62927/finalize_review
I see:
![bildschirmfoto 2017-12-20 um 16 17 11](https://user-images.githubusercontent.com/612992/34213985-4da1f28a-e5a1-11e7-9496-c2475ad0b2a6.png)


I hope that this helps you. I cannot at the moment do any editing and finalizing because I cannot see any text / XML version of the file.

**Comment by jcowey** [comment]
The text in this entry file is related to 
http://papyri.info/hgv/699551

In the repo for o.bankes;;
https://github.com/papyri/idp.data/tree/master/DDB_EpiDoc_XML/o.bankes

as you will see, there is no entry for 
https://github.com/papyri/idp.data/blob/master/DDB_EpiDoc_XML/o.bankes/o.bankes.77.xml

which is would the file to be finalized would be renamed to

**Comment by ryanfb** [comment]
It looks like the problem with this one was that the rename went through in Git, but something caused the rename to be rolled back from the database, leaving it in an inconsistent state. I've updated the database to point to the correct identifier and re-sent the publication for finalization again at: http://papyri.info/editor/publications/62930
